src/app/cli/src/init/client.ml

```diff
@@ -1105,19 +1105,22 @@ let generate_libp2p_keypair =
        @@ fun () ->
        Deferred.ignore
          (let open Deferred.Let_syntax in
-         let logger = Logger.create () in
+         (* FIXME: I'd like to accumulate messages into this logger and only dump them out in failure paths. *)
```

Comment:
Can we open an issue to add this feature to the logger?

Reply:
yes.